xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/XWiki/QuickSearchUIX.xml

```diff
@@ -143,8 +143,9 @@
         // We close the global search only if the focus is not on the search input or the search button.
         // Without this, the global search would be close each time the user click on the button (even when it's for
         // actually performing the search).
-        if (document.activeElement !== globalSearchButton[0] &amp;&amp; document.activeElement !== globalSearchInput[0]) {
+        if (!document.getElementById('globalsearch').contains(document.activeElement)) {
           globalSearch.addClass('globalsearch-close');
+          document.fire("xwiki:suggest:collapsed",globalSearchInput);
```

Comment:
We're not supposed to use Prototype.js API any more, see http://api.prototypejs.org/dom/document/fire/index.html , especially inside a require block that gets jQuery injected. See https://api.jquery.com/trigger/#trigger-eventType-extraParameters .